spec/lib/cyclonedx/report_podfile_lock_cyclonedx_spec.rb

```diff
@@ -0,0 +1,67 @@
+require_relative '../../spec_helper'
+require 'json'
+
+describe Cyclonedx::ReportPodfileLock do
+  describe "#run" do
+    it 'should report all the deps in the Podfile.lock' do
+      repo = Salus::Repo.new('spec/fixtures/report_podfile_lock/normal')
+      scanner = Salus::Scanners::ReportPodfileLock.new(repository: repo, config: {})
+      scanner.run
+
+      maven_cyclonedx = Cyclonedx::ReportPodfileLock.new(scanner.report)
+      expect(maven_cyclonedx.build_components_object).to match_array(
+        [
+          {
+            "bom-ref": "pkg:cocoa/boost-for-react-native@1.63.0",
+            type: "library",
+            group: "",
+            name: "boost-for-react-native",
+            version: '1.63.0',
+            purl: "pkg:cocoa/boost-for-react-native@1.63.0",
+            properties: [
+              { key: "source", value: "" },
```

Comment:
It would be nice to support the explicit source when given:

pod "Brightcove-Player-FreeWheel", :git => 'https://github.com/brightcove/BrightcoveSpecs.git'

Having that map to source would be a nice feature